lib/archethic/bootstrap/network_init.ex

```diff
@@ -84,6 +84,8 @@ defmodule Archethic.Bootstrap.NetworkInit do
     tx
     |> self_validation()
     |> self_replication()
+
+    SharedSecrets.persist_gen_addr(tx.address, :node_shared_secrets)
```

Comment:
I think you don't need to persist the term here as you do it after the bootstrap (function post_bootstrap is always call even after network init)